Server/Plugins/APIDump/Hooks/OnPlayerEditingBook.lua

```diff
@@ -0,0 +1,27 @@
+return
+{
+	HOOK_PLAYER_EDITING_BOOK =
+	{
+		CalledWhen = "A player is editing a book.",
+		DefaultFnName = "OnPlayerEditingBook",  -- also used as pagename
+		Desc = [[
+			This hook is called whenever a {{cPlayer|player}} is editing a book.
+			See also the {{OnPlayerEditingBook|HOOK_PLAYER_EDITED_BOOK}} hook for a similar hook, is called when a
+			player has edited a book.
+		]],
+		Params =
+		{
+			{ Name = "Player", Type = "{{cPlayer}}", Notes = "The player that is editing the book" },
+			{ Name = "BookContent", Type = "{{cBookContent}}", Notes = "The class that contains the current info of the book" },
```

Comment:
Is this before or after the change? Can we get both, somehow? Ideally so that a plugin can modify the new content.